specifyweb/frontend/js_src/lib/components/UserPreferences/ApplyPreferences.tsx

```diff
@@ -100,28 +100,28 @@ export function SetCssVariables(): null {
       saveButtonColor: getUserPref('general', 'appearance', 'saveButtonColor'),
       secondaryButtonColor: getUserPref(
         'general',
-        'appearance',
+        'buttonColor',
```

Comment:
consider renaming this to `button` to match the UI label and to be more general in case we add non-color button customization options in the future (and like you mentioned, to remove redundancy with the item name - as each name includes "color" in it)